tests/functional/adapter/store_failures/test_store_failures.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+from dbt.tests.util import run_dbt
+
+from tests.functional.adapter.store_failures.fixtures import (
+    seed_csv,
+    table_model,
+    table_profile_yml,
+)
+
+
+class TestStoreFailuresBase:
```

Comment:
No need to have base class here. You can include all methods in one class.

Reply:
done